src/main/java/ui/components/NavigableListView.java

```diff
@@ -111,25 +114,46 @@ public void restoreSelection() {
                 selectedIndex = Optional.of(nextIndex);
 
                 // The next item will be considered selected
-                onItemSelected.accept(nextIndex);
+                triggerItemSelected(nextIndex);
             }
         }
     }
 
+    /**
+     * This method triggers the callback for onItemSelected with the index and "false" parameter indicating that
+     * it is called for left clicks, key presses etc but not right clicks.
+     * The boolean parameter enables us to perform different actions
+     * (e.g. deciding whether the bView should be loaded with that issue) based on different kinds of inputs
+     */
```

Comment:
This comment sounds like you are deliberately trying to perplex the reader. :-|
It's a simple method that needs a simple explanation.
`Triggers the 'selected' action on the card at index.`